packages/ui/primitives/document-flow/field-item.tsx

```diff
@@ -89,12 +117,25 @@ export const FieldItem = ({
     };
   }, [calculateCoords]);
 
+  const handleOnClick = (event: MouseEvent) => {
+    if (settingsActive && cardRef.current && !event.composedPath().includes(cardRef.current)) {
+      setSettingsActive(false);
+    }
+  };
+
+  useEffect(() => {
+    document.body.addEventListener('click', handleOnClick);
+    return () => {
+      document.body.removeEventListener('click', handleOnClick);
+    };
+  }, [settingsActive]);
+
   return createPortal(
     <Rnd
       key={coords.pageX + coords.pageY + coords.pageHeight + coords.pageWidth}
-      className={cn('z-20', {
-        'pointer-events-none': passive,
-        'pointer-events-none opacity-75': disabled,
+      className={cn('group z-20', {
+        'active:pointer-events-none': passive,
```

Comment:
couldn't think of a better way of making the hover work on `passive`/`disabled` fields. happy to hear suggestions